components/camel-rest-swagger/src/main/java/org/apache/camel/component/rest/swagger/RestSwaggerEndpoint.java

```diff
@@ -165,7 +177,18 @@ public Consumer createConsumer(final Processor processor) throws Exception {
     @Override
     public Producer createProducer() throws Exception {
         final CamelContext camelContext = getCamelContext();
-        final Swagger swagger = loadSpecificationFrom(camelContext, specificationUri);
+        SSLContextParameters sslContextParameters = getSslContextParameters();
+        if (sslContextParameters == null) {
```

Comment:
IMHO, I'd move `sslContextParameters` checks to a separate function and I'd prefer to use `if else` only in this context in order to avoid the `null` checks in case is already assigned

Reply:
I don't have a problem with suggested approach, I'll change it.